samples/nrf9160/lwm2m_client/location_assistance.rst

```diff
@@ -4,30 +4,41 @@ Location assistance
 ###################
 
 The LwM2M Client sample supports a proprietary mechanism to fetch location assistance data from `nRF Cloud`_ by proxying it through the LwM2M server.
-This is achieved by using LwM2M objects ECID-Signal Measurement Information object(ID 10256) and proprietary Location assistance object (ID 50001).
+The location assistance is divided to two distinct approaches, GNSS-based and cell-based estimate, which is called a ground fix.
+The approaches use different set of LwM2M objects. The GNSS-based approach uses only the GNSS Assistance object (ID 33625).
+The ground fix approach uses mandatory Ground Fix Location object (ID 33626) and optional ECID-Signal Measurement Information object (ID 10256) and Visible Wi-Fi Access Point object (33627).
 
 To trigger location assistance, use the following buttons on the board:
 
 * **Button 1** - For GNSS
-* **Button 2** - For cell location
+* **Button 2** - For ground fix
 
 .. note::
    This feature is currently under development and as of now, only AVSystem's Coiote LwM2M server can be used for utilizing the location assistance data from nRF Cloud.
 
-There are two different methods of obtaining the location assistance:
+.. note::
+   The old Location Assistance object (ID 50001) will be removed in the future and shouldn't be used anymore.
```

Comment:
Why don't we remove it now?
Before the next NCS release, this is going to be tested, and server side should have the support for new object implemented. There is no use for old object anymore.

Reply:
Let me have a look and see how much change it will require. Since 50001 is not deprecated, but actually removed in this PR the ATv2 should probably be updated as well. I'm assuming that location assistance won't work after this PR gets in without the change?

Reply:
Ill create a PR towards you branch if that's fine with you